core-plugins/docs/Excel-batchsource.md

```diff
@@ -34,12 +34,12 @@ a new table or existing one. (Macro-enabled)
 
 **reprocess:** Specify whether the files mentioned in the memory table should be reprocessed or not.
 
-**sheet:** Specifies whether sheet has to be processed by sheet name or sheet no.
+**sheet:** Specifies whether sheet has to be processed by sheet name or sheet number.
 
 **sheetValue:** Specifies the value corresponding to 'sheet' input. Value can be either actual
-sheet name or sheet no.
-for example: 'Sheet1' or '1' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet'
-input respectively. (Macro-enabled)
+sheet name or sheet number.
+for example: 'Sheet1' or '0' in case user selects 'Sheet Name' or 'Sheet Number' as 'sheet'
+input respectively. Sheet number starts with 0. (Macro-enabled)
```

Comment:
In Excel POI api, sheet number starts at 0. Therefore the implementation is done based on the sheet number starting with 0.